clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -5833,6 +5840,15 @@ void Clang::ConstructJob(Compilation &C, const JobAction &JA,
         CmdArgs.push_back("-fno-sycl-esimd-build-host-code");
     }
 
+    if (IsSYCLCUDACompat) {
+      Args.addOptInFlag(CmdArgs, options::OPT_fsycl_cuda_compat,
+                        options::OPT_fno_sycl_cuda_compat);
+      // clang's CUDA headers require this ...
+      CmdArgs.push_back("-fdeclspec");
```

Comment:
There is currently a FIXME in `clang/include/clang/Driver/Options.td` regarding the use of `declspec` in `clang/lib/Headers/__clang_cuda_builtin_vars.h`. Perhaps elevate this comment to a FIXME as well since this is effectively a workaround.

Reply:
changed into a  FIXME